tests/test_api.py

```diff
@@ -643,87 +613,108 @@ def  test_length_and_hash_validation(self):
             # test exceptions
             expected_length = snapshot_metafile.length
             snapshot_metafile.length = 2345
-            self.assertRaises(exceptions.LengthOrHashMismatchError,
-                snapshot_metafile.verify_length_and_hashes, data)
+            self.assertRaises(
+                exceptions.LengthOrHashMismatchError,
+                snapshot_metafile.verify_length_and_hashes,
+                data,
+            )
```

Comment:
It think these are more evidence of how we should ban `self.assertRaises()` unless it's used in a `with self.assertRaises():`.

however, I'm fine with this PR being a mostly "machine-generated" change so that's just a side note.

Reply:
I created an issue about that #1660.